website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,219 @@
+---
+author: Ayush Sharma
+authorURL: https://twitter.com/ayusharma_
+authorFBID: 1122901551
+title: Verdaccio 4 released today !!!
+---
+
+#### Release name: Freedom
+
+Verdaccio is a free open source javascript package proxy registry. It is fully compatible with pnpm, yarn and npm package management clients. It follows the CommonJS compliant package specifications. 
+
+You can install and upgrade to the latest version by following commands:
+using **npm** 
+````
+npm install -g verdaccio@4.0.0
+````
+
+or using **Yarn** 
+````
+yarn global add verdaccio@4.0.0
+````
+
+or using **pnpm** 
+````
+yarn global add verdaccio@4.0.0
```

Comment:
To install Verdaccio globally with pnpm should be `pnpm install -g verdaccio@4.0.0`